test/unit/network_spec.js

```diff
@@ -176,4 +176,62 @@ describe('network', function() {
       done.fail(reason);
     });
   });
+
+  describe('PDFNetworkStreamFullRequestReader_extractNameFrom', function () {
+    var stream = new PDFNetworkStream({
+      source: {
+        url: pdf1,
+        rangeChunkSize: 65536,
+        disableStream: true,
+      },
+      disableRange: true,
+    });
+
+    var requestReader = stream.getFullReader();
+
+    it('returns null when no content disposition', function() {
+      expect(requestReader._extractNameFrom(null)).toBeNull();
+      expect(requestReader._extractNameFrom(undefined)).toBeNull();
+    });
+
+    it('Response header for the main body', function () {
+      expect(requestReader._extractNameFrom(
+        'Content-Disposition: inline'
+      )).toBeNull();
+
+      expect(requestReader._extractNameFrom(
+        'Content-Disposition: attachment'
+      )).toBeNull();
+
+      expect(requestReader._extractNameFrom(
+        'Content-Disposition: attachment; filename="filename.pdf"'
+      )).toBe('filename.pdf');
+    });
+
+    it('Header of multipart body', function () {
+      expect(requestReader._extractNameFrom(
+        'Content-Disposition: form-data'
+      )).toBeNull();
+
+      expect(requestReader._extractNameFrom(
+        'Content-Disposition: form-data; name="fieldName"'
```

Comment:
Could you add a test case where `name="filename"`, just to make sure that the regex does not pick that up as a valid filename?

Reply:
Good point. I'll do